src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -931,18 +920,16 @@ def process_question_pl(
             question_html += f"{converter(part,parsed_q,data2)}"
 
         if parsed_q["num_parts"] > 1:
-            question_html += "</div>\n</div>\n"
+            question_html += "</pl-card>\n"
 
         # Add pl-submission-panel and pl-answer-panel (if they exist)
         subm_panel = parsed_q["body_parts_split"][part].get("pl-submission-panel", None)
         q_panel = parsed_q["body_parts_split"][part].get("pl-answer-panel", None)
 
         if subm_panel:
-            question_html += (
-                f"\n<pl-submission-panel>{ subm_panel } </pl-submission-panel>\n"
-            )
+            question_html += f"\n<pl-submission-panel>{subm_panel}</pl-submission-panel>\n"
         if q_panel:
-            question_html += f"\n<pl-answer-panel>{ q_panel } </pl-answer-panel>\n"
+            question_html += f"\n<pl-answer-panel>{q_panel}</pl-answer-panel>\n"
 
         # TODO: Add support for other panels here as well !
```

Comment:
I wasn't sure, so I didn't touch the comment, but I think that TODO is safe to remove since the other decorative/conditional elements don't need to have first-class support in markdown